src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -323,3 +352,11 @@ bool EquationTextBox::ShouldDeleteButtonBeVisible()
     }
     return (!text->IsEmpty() && m_HasFocus);
 }
+
+void EquationTextBox::OnRichEditMenuOpening(Platform::Object ^ sender, Platform::Object ^ args)
+{
+    if (m_kgfEquationMenuItem != nullptr)
+    {
+        m_kgfEquationMenuItem->IsEnabled = EquationTextBox::RichEditHasContent();
```

Comment:
This is kind of intentional because I think having disappearing UI in this case can be confusing as there isn't really any other UI hinting at what should be shown and what shouldn't.